File: src/i18n/es_ar.go
package i18n

// esAROverrides is empty on top of esLatamOverrides.
//
// What makes Argentine Spanish immediately recognisable is voseo — seleccioná
// where Spain writes selecciona — and voseo is an alternative to tuteo, to the
// familiar tú. This catalog never uses it: it is written the way error messages
// are, infinitives for the audit log ("crear cuenta") and impersonal statements
// for the rest ("la partición ya existe"). It never addresses the reader at
// all, so there is no familiar form here for voseo to replace. The frontend
// does address the reader, but in the formal usted register, which is identical
// in Buenos Aires and Madrid — so es-AR.js is empty for the same reason.
var esAROverrides = map[string]string{}

// esARMessages is es-ES by way of the shared American departures.
var esARMessages = derive(esLatamMessages, esAROverrides)

File: ui/src/i18n/es-AR.js
import derive from './derive.js'
import esLatam from './es-latam.js'

/**
 * Empty on top of es-latam.js.
 *
 * What makes Argentine Spanish immediately recognisable is voseo —
 * *seleccioná* where Spain writes *selecciona*. But voseo is an alternative to
 * *tuteo*, and this catalog never uses *tú*: es-ES.js addresses the reader as
 * *usted* (`Seleccione`, `Consulte`), and *usted* is identical in Buenos Aires
 * and Madrid. There is no second-person familiar form here for voseo to
 * replace.
 *
 * Writing voseo anyway would mean first switching the whole catalog from formal
 * to familiar address — a register change dressed up as a locale, and one that
 * would make es-AR the only Spanish that speaks to the reader differently.
 */
export const esAROverrides = {}

/** Spanish (Argentina) — es-ES by way of the shared American departures. */
const esAR = derive(esLatam, esAROverrides)

export default esAR
